src/AWSS3.jl

```diff
@@ -981,7 +1004,7 @@ function s3_put(
     return parse_response ? parse(response) : response
 end
 
-s3_put(a...; b...) = s3_put(global_aws_config(), a...; b...)
+s3_put(args...; kwargs...) = s3_put(global_aws_config(), args...; kwargs...)
```

Comment:
```suggestion
function s3_put(bucket, path, data::Union{String,Vector{UInt8}}, data_type="", encoding=""; kwargs...
  return s3_put(global_aws_config(), bucket, path, data, data_type, encoding; kwargs...)
end
```

Reply:
no longer valid (followed the below @omus style)